index.html

```diff
@@ -0,0 +1,130 @@
+                </p>
+                <h2 class="time">2023.10.15(THU) ~ 16(FRI)</h2>
+                <p>@ National Museum of Nigeria, Art Center Lagos and more</p>
+            </article>
+        </section>
+    </header>
+    <section id="main-program">
+        <h2 class="program-title wht">Main Program</h2>
+        <section id="cards">
+            <article class="card">
+                <img src="./assets/icons/home_iot_device_FILL0_wght400_GRAD0_opsz48.svg" alt="home_iot_device">
+                <h2>Home IoT</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/smart_outlet_FILL0_wght400_GRAD0_opsz48.svg" alt="smart_outlet">
+                <h2>Smart Outlet</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/assistant_on_hub_FILL0_wght400_GRAD0_opsz48.svg" alt="home_assistance">
+                <h2>Home Assistance</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+            <article class="card">
+                <img src="./assets/icons/rocket_launch_FILL0_wght400_GRAD0_opsz48.svg" alt="rocket_science">
+                <h2>Rocket Science</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos accusantium ipsam minima.
+                </p>
+            </article>
+        </section>
+        <h2 class="Program-foot dk">SEE THE WHOLE PROGRAM</h2>
+        <h2 class="Program-foot mb">Join CC Global Summit 2023</h2>
+    </section>
+    <section id="speakers">
+
+        <h2 class="program-title">Featured Speakers</h2>
+        <section id="speaker"></section>
+        <button id="toggle-more" class="toggle-more-less mb">MORE</button>
+
+    </section>
+    <footer>
+        <section id="sponsors">
+            <h2 class="program-title wht">Partner</h2>
+            <ul class="sponsors">
+                <li>mosilla</li>
+                <li><span>Goog<span class="bd-1">l</span>e</span></li>
+                <li>NAVER</li>
+                <li>daumkakao</li>
+                <li>airbnb</li>
+            </ul>
+        </section>
+        <section class="buttom-f-nav">
+            <a class="logo logo-abt" href="#">BLK-NG</a>
+            <article class="short-f-note">
+                <h2>2023 Creative Commons Korea Some Rights Reserved.</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Modi.
```

Comment:
- [ ] 👋 Thank you for your work on the project! However, I noticed that this elements are not styled according to the design requirements. To improve the visual consistency, please follow the design and update the colors for those elements in your code. It's essential to ensure that all elements align with the design to create a cohesive and polished user interface.

Keep up the good work, and don't forget to make those necessary style adjustments! If you have any questions, feel free to ask. Happy coding! 🌟💻🚀